sirepo/tornado.py

```diff
@@ -6,8 +6,30 @@
 """
 from pykern.pkdebug import pkdlog, pkdexc
 import sirepo.util
+import tornado.locks
 import tornado.queues
 
+class Event(tornado.locks.Event):
```

Comment:
document why this is here

Reply:
https://github.com/radiasoft/sirepo/pull/2731/commits/6960d7d6b3672704a51babc454d7c6b07a162c87